install/kubernetes/helm/istio/charts/gateways/templates/deployment.yaml

```diff
@@ -236,6 +236,8 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
+          - name: SDS_ENABLED
+            value: "{{ $.Values.global.sds.enabled }}"
```

Comment:
note to other reviewers - Values.global.sds.enabled is defined and defaults to `false` in the top level `values.yaml` file.

Reply:
Yes, SDS is alpha feature. It is disabled by default.

Reply:
Can you flip the flags in the SDS configs:
https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/values-istio-sds-auth.yaml#L2
https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/example-values/values-istio-googleca.yaml#L2

Reply:
Done.